--- sources/shiboken2/ApiExtractor/CMakeLists.txt	2020-08-26 17:37:19.411334400 +0300
+++ sources/shiboken2/ApiExtractor/CMakeLists.txt	2020-08-26 17:36:55.459967400 +0300
@@ -84,7 +84,7 @@
 
 target_compile_definitions(apiextractor PRIVATE CMAKE_CXX_COMPILER="${CMAKE_CXX_COMPILER}")
 
-set(LIB_INSTALL_DIR "${CMAKE_INSTALL_PREFIX}/lib${LIB_SUFFIX}" CACHE PATH "The subdirectory relative to the install prefix where libraries will be installed (default is /lib${LIB_SUFFIX})" FORCE)
+set(LIB_INSTALL_DIR "lib${LIB_SUFFIX}" CACHE PATH "The subdirectory relative to the install prefix where libraries will be installed (default is /lib${LIB_SUFFIX})" FORCE)
 
 if (BUILD_TESTS)
     find_package(Qt5Test 5.12 REQUIRED)
