--- pyside-setup-everywhere-src-5.12.1/sources/pyside2/CMakeLists.txt	2019-01-31 15:05:01.000000000 +0100
+++ patch/CMakeLists.txt	2019-02-17 17:31:41.045272590 +0100
@@ -354,7 +354,8 @@
 
 # Collect all optional modules.
 set(ALL_OPTIONAL_MODULES Xml XmlPatterns Help Multimedia MultimediaWidgets OpenGL Positioning Location Qml Quick QuickWidgets Scxml Script ScriptTools Sensors TextToSpeech Charts Svg DataVisualization)
-find_package(Qt5UiTools)
+#find_package(Qt5UiTools)
+set(Qt5UiTools_FOUND FALSE)
 if(Qt5UiTools_FOUND)
     list(APPEND ALL_OPTIONAL_MODULES UiTools)
 else()
@@ -497,7 +498,8 @@
 endif()
 
 add_subdirectory(libpyside)
-find_package(Qt5Designer)
+#find_package(Qt5Designer)
+set(Qt5Designer_FOUND FALSE)
 if(Qt5UiTools_FOUND AND Qt5Designer_FOUND)
     add_subdirectory(plugins)
 endif()
